frontend/src/features/idir/search/components/IDIRPermitSearchRowActions.tsx

```diff
@@ -136,6 +139,31 @@ export const IDIRPermitSearchRowActions = ({
     }
   };
 
+  const handleResend = async (
+    permitId: string,
+    email: string,
+    fax?: Nullable<string>,
+  ) => {
+    const response = await resendPermitMutation.mutateAsync({
+      permitId,
+      email,
+      fax,
+    });
+
+    if (response.status === 201) {
+      setOpenResendDialog(false);
+      setSnackBar({
+        showSnackbar: true,
+        setShowSnackbar: () => true,
+        message: "Successfully sent",
+        alertType: "success",
+      });
+    } else {
+      setOpenResendDialog(false);
```

Comment:
Could relocate `setOpenResendDialog` outside of the if-else block since it's gonna be called either way.

Reply:
Good point, changed